en/docs/services/create-your-first-service.md

```diff
@@ -1,177 +1,260 @@
 
-Now that you have verified that the `covid-stats` service works as expected, you can deploy it.
 
-## Step 4: Deploy the service
+## Step 7: Deploy the service
 
-To deploy the `covid-stats` service, follow this procedure:
+To deploy the `CovidStatus` service, follow these steps:
 
 1. Click the **Go Live** icon in the left pane.
 
-    ![Test Icon](../assets/img/services/deploy-icon.png){.cInlineImage-bordered}
+    ![Deploy icon](../assets/img/services/deploy-icon.png){.cInlineImage-bordered}
 
 2. To deploy the service, click **Deploy**.
 
-    The status of the service changes to **Deployed**, and the following logs show that the service is successfully deployed:    
-
-    ![Deployment logs](../assets/img/services/deployment-logs.png){.cInlineImage-full}
+    The status of the service changes to **Deployed**, and the following logs appear to indicate that the product is successfully deployed.    
 
+    ![Deployment logs](../assets/img/services/view-deployment-logs.png){.cInlineImage-full}
+    
+    In addition, the following message appears:
+    
+    ```
+    Deployed successfully. Note that the service will automatically undeploy in 12 hours.
+    ```
+   
 3. Once the service is deployed, click on **://cURL**.
 
-    ![Get cURL command](../assets/img/services/copy-curl-command.png){.cInlineImage-full}
+    ![Get cURL command](../assets/img/services/get-curl-command.png){.cInlineImage-full}
     
     Copy the cURL command that is displayed.
     
-4. Invoke the `covid-stats` service by issuing a few cURL commands using the Postman application. The responses are logged as shown below:
-    
-    ![covid-stats-log](../assets/img/services/covid-stats-log.png){.cInlineImage-half}
-    
-5. On the Choreo Console **Go Live** tab, the requests are logged as follows:
-    
-    ![Execution History](../assets/img/services/execution-history.png){.cInlineImage-half}
-    
-Now you are ready to observe the `covid-stats` service based on the statistics generated as a result of the cURL commands you issued. 
+    !!! tip
+        The cURL command will be something as given in the example below:<br/><br/>
+        `curl "https://johndoe.dv.choreoapis.dev/covidstatus/1.0.0/stats/{country}" -H 'API-Key: <API_KEY>>' -X GET<br/><br/>
+        The value for the **country** parameter can be changed as required.
+        
+4. Invoke the service a few times via the terminal by issuing the cURL command you copied.
+      
+Now you are ready to observe the `CovidStatus` service based on the statistics generated as a result of the cURL commands you issued. 
 
-## Step 5: Observe the service
+## Step 8: Observe the service
 
-To observe the `covid-stats` service, click the **Observe** icon in the left panel.
+To observe the `CovidStatus` service, click the **Observe** icon in the left panel.
 
 ![Observe Icon](../assets/img/services/observe-icon.png){.cInlineImage-bordered}
```

Comment:
Need to follow sentence-style for alt text.

Reply:
Corrected